__tests__/integration/nanocontracts/bet.test.ts

```diff
@@ -478,7 +478,7 @@ describe('full cycle of bet nano contract', () => {
 
     jest.spyOn(wallet.storage, 'processHistory');
     expect(wallet.storage.processHistory.mock.calls.length).toBe(0);
-    await waitNextBlock(wallet.storage);
```

Comment:
chore: We should just remove the `waitNextBlock` import from the file, as it's not being used anymore.

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/825/commits/a4827652e34169f0a1ecd2b7d284cde02dbb8bf3